README.md

```diff
@@ -1 +1,196 @@
+  "last_name": "Sanchez",
+  "email": "picklerick@gmail.com"
+}
+```
+
+**Step 3 -** use data tag to indicate wherein the onboarding process the user is and when they got there. If our process has only two steps, we would choose either `onboarding_started` or `onboarding_complete` and mark the time for each step.
+
+```json
+{
+  "last_site_visit_date": "",
+  "first_name": "Rick",
+  "last_name": "Sanchez",
+  "email": "picklerick@gmail.com",
+  "onboarding_stage": "started",
+  "onboarding_start_dt: "1638209022859" // 'Mon Nov 29 2021 12:04:00 GMT-0600 (Central Standard Time)'
+}
+```
+
+**Step 4 -** N/A
+
+## Segments
+
+Segments allow you to group devices based on their attributes. We can take advantage of one of the features of segments – sending an automated messages when a user/devices moves into a segment – to deliver an event-triggered notication workflow.
+
+### How to create a segment
+
+1. Navigate to _Audience_.
+2. Click _New Segment_ button.
+3. Give the segment a name.
+4. Add rules for how users will be filtered.
+   <img width="1439" alt="Screen Shot 2021-10-12 at 1 37 57 PM" src="https://user-images.githubusercontent.com/1715082/137010953-d2c9d0e2-1823-4327-bcd2-2dcfdcc80fe8.png">
+5. Click the _Create Segment_ button.
+
+### How to inspect data tags
+
+Since segments depend on data tags, we must first learn how to inspect their values.
+
+1. Navigate to _Audience_
+   <img width="1437" alt="audience-menu-selection" src="https://user-images.githubusercontent.com/1715082/137007996-c1c20dc2-e649-4f78-8b38-8c8388d12f34.png">
+2. Select _All Users_ from the submenu
+   <img width="1441" alt="users-submenu-selection" src="https://user-images.githubusercontent.com/1715082/137008242-8b0c87e8-1f14-4eae-84bb-7c01ffea3f03.png">
+3. Adjust the _Displayed columns_ to see your tags
+   <img width="1440" alt="displayed columns" src="https://user-images.githubusercontent.com/1715082/137007643-9c3993db-c8ee-4737-bed6-e03bf0d80e2a.png">
+
+## Message Templates
+
+[Message templates](https://documentation.onesignal.com/docs/templates) enable you to scaffold out a message whose content is dynamically generated via data tag substitution.
+
+### How to creating a message template
+
+We can create a message template using the dashboard by following these steps.
+
+1. Navigate to _Messages_
+2. Select _Templates_ from the submenu
+3. Click the _New Push Template_ button
+4. Complete the _Message_ form and save
+   <img width="1438" alt="message template" src="https://user-images.githubusercontent.com/1715082/137006870-da9066f1-1140-4acb-9eee-f1b5903a23b5.png">
+
+## Automated Messages
+
+[Automated messages](https://documentation.onesignal.com/docs/automated-messages) enable you to push notifications or emails when a user enters a segment and are a key building block for implementing event-triggered notifications.
+
+### How to create an automated message
+
+We can create automated messages using The OneSignal Dashboard by following these steps.
```

Comment:
I think "the OneSignal dashboard" might be more appropriate so as to not imply some product subbranding.  See Capitalization Rules in the Content Style Guide